src/content/en/2021/cdn.md

```diff
@@ -96,26 +97,28 @@ With the evolution of CDN technology an expressway was set up on the internet fo
+The impact of this can be seen in the above chart when we compare this against the same data in [Web Almanac 2019 - CDN Chapter](../2019/cdn) (there was no CDN Chapter in Web Almanac 2020). It's good to see the trend of HTMLs using CDN has improved by 7% between 2019 and 2021.  This shows that more of the industry is leveraging CDNs to take benefit of consistent content delivery times and minimize the impact of congestion on Internet.
 
 Looking at third party content, there is negative growth for CDN adoption. Compared to [Web Almanac 2019 - CDN Chapter](../2019/cdn), we see 3% reduction in domains using CDNs. Third party domains are used by SaaS vendors for analytics, advertisements, responsive pages, etc. It is in the SaaS vendor's interest to use CDNs for their services. Their content is used by multiple web owners and this content gets accessed by end users across geographies, making CDNs necessary from both a business and performance standpoint. This is evident in the charts where it's clear that third party content has the highest adoption of CDN.
 
 But why do we see this negative growth in CDN Adoption for third party domains?
 
 The probable reasons for this may include:
 
-* The HTTP/2 protocol requires web owners to consolidate the domains instead of using multiple domains for optimal performance.
-* Contribution of third party content to total page weight has also increased over the years (refer to the [Third Parties](./third-parties) chapter for more details) leading to increased page load time concerns for web owners.
-* Customization/personalization of third party scripts to suit the requirements of web owners.
+* The HTTP/2 protocol requires web owners to consolidate the domains instead of using multiple domains for optimal performance
+* Contribution of third party content to total page weight has also increased over the years (refer to the [Third Parties](./third-parties) chapter for more details) leading to increased page load time concerns for web owners
+* Customization/personalization of third party scripts to suit the requirements of web owners
 
 These changes have led to the SaaS vendors offering "self-hosting" options to web owners. This leads to more content being delivered over the first party domain instead of the vendor's domain. When this happens, it's up to the web owner to either deliver the content over a CDN or directly from their hosting infrastructure.
 
-While we observed CDN adoption across different types of content, we will not look at this data a bit different below
+While we observed CDN adoption across different types of content, we will not look at this data a bit different below.
+
+{# TODO: I don't get what - "we will not look at this data a bit different below" means exactly #}
```

Comment:
@Navaneeth-akam

Reply:
Ahhh....a bit of typo here. It should have been "we will look at this data a bit different below"